### PR TITLE
hint at dormant influxd

### PR DESCRIPTION
**Pull request by tpltnt**
Hi InfluxDB people,

i added a hint on the influx CLI interface which indicates that influxd needs to be started.
Cheers,
tpltnt

### CODE REVIEW

File: content/docs/v0.9/introduction/getting_started.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ InfluxDB shell 0.9.0
 
 > **Note:** The InfluxDB HTTP API runs on port `8086` by default. Therefore, `influx` will connect to port `8086` and `localhost` by default. If you need to alter these defaults run `influx --help` or read the [man page](../clients/shell.html)
 
+> **Note:** If the "Connected to http://..." part is missing, then `influxd` is not running and needs to be started.
```
Comment (otoolep) [inline content/docs/v0.9/introduction/getting_started.md]: I don't really follow you. Can you demonstrate the problem you encountered?

**Comment by tpltnt** [comment]
@otoolep here is an example:

*without* `ìnfluxd` running:
```
Connected to http://localhost:8086 version 
InfluxDB shell 0.9
> 
```
* *with* `influxd` running
```
Connected to http://localhost:8086 version 0.9
InfluxDB shell 0.9
>
```
When `influxd` is running, the version string "0.9" is printed.

**Comment by otoolep** [comment]
That should not happen with the latest code, due to a recent fix. With the latest code is `influxd` is not running I get:

```
./influx 
Failed to connect to http://localhost:8086
Please check your connection settings and ensure 'influxd' is running.
```

**Comment by otoolep** [comment]
Thanks for the change @tpltnt -- but master code no longer suffers from this issue, so there is no need to update the docs.

Please let me know if this is not what you see.

**Pull request closed without merging**